src/modules/business/infra/http/controllers/BusinessContactController.ts

```diff
@@ -0,0 +1,83 @@
+import { Request, Response } from 'express';
+
+import { container } from 'tsyringe';
+
+import CreateBusinessContactService from '@modules/business/services/businessContact/CreateBusinessContactService';
+import DeleteBusinessContactService from '@modules/business/services/businessContact/DeleteBusinessContactService';
+import ShowBusinessContactService from '@modules/business/services/businessContact/ShowBusinessContactService';
+import UpdateBusinessContactService from '@modules/business/services/businessContact/UpdateBusinessContactService';
+import IndexBusinessContactService from '@modules/business/services/businessContact/IndexBusinessContactService';
+
+export default class BusinessContactController {
+  public async create(request: Request, response: Response): Promise<Response> {
```

Comment:
it is very likely that you have to separate the create, delete and update method in a file of specific routes, as they hold the need for the business to be logged into the application